tests/foreman/destructive/test_clone.py

```diff
@@ -0,0 +1,111 @@
+"""Tests for satellite-maintain clone
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: SatelliteClone
+
+:Assignee: lpramuk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.hosts import Satellite
+
+SSH_PASS = settings.server.ssh_password
+pytestmark = pytest.mark.destructive
+
+
+@pytest.mark.parametrize("sat_ready_rhel", [8], indirect=True)
+@pytest.mark.parametrize('backup_type', ['online', 'offline'])
+@pytest.mark.parametrize('skip_pulp_content', [True, False], ids=['skip_pulp', 'no_skip_pulp'])
+def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pulp_content):
+    """Make an online/offline backup with/without pulp data of Satellite and clone it (restore it).
+
+    :id: 5b9182d5-6789-4d2c-bcc3-6641b96ab277
+
+    :steps:
+        1. Create an online/offline backup with/without pulp data
+        2. Copy it to target rhel
+        3. Install satellite-clone
+        4. Run satellite clone on target rhel
+
+    :expectedresult:
+        1. Satellite-clone ran successfully
+
+    :parametrized: yes
+    """
+    rhel_version = sat_ready_rhel._v_major
+    sat_version = target_sat.version
+
+    # SATELLITE PART - SOURCE SERVER
+    # Enabling and starting services
+    assert target_sat.cli.Service.enable().status == 0
+    assert target_sat.cli.Service.start().status == 0
+
+    # Doing backup
+    backup_result = target_sat.cli.Backup.run_backup(
+        backup_dir='/var/backup',
+        backup_type=backup_type,
+        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': skip_pulp_content},
+    )
```

Comment:
Are you planning to include any scenario for restoring pulp content? like sync a few repos and verify they exist after clone/restore, will suggest checking `tests/foreman/maintain/test_backup_restore.py::test_positive_backup_restore`

Reply:
In the future probably yes but not now.